sonar-project.properties

```diff
@@ -2,7 +2,7 @@ sonar.projectKey=open-cluster-management_console
 sonar.projectName=console
 sonar.organization=open-cluster-management
 sonar.sources=frontend/src,backend/src
-sonar.exclusions=node_modules/**/*,frontend/node_modules/**/*,frontend/src/atoms.tsx, frontend/src/lib/nock-util.ts, frontend/src/ui-components/**/*.stories.tsx
+sonar.exclusions=node_modules/**/*,frontend/node_modules/**/*,frontend/src/atoms.tsx,frontend/src/lib/nock-util.ts,frontend/src/ui-components/**/*.stories.tsx,frontend/src/routes/Home/Search/search-sdk/search-sdk.ts
```

Comment:
Should we really permanently exclude this file from Sonar scanning? Is this a test coverage issue or something else?

Reply:
My thinking was that this file is auto-generated from graphql-codegen. The coverage is low on this file mainly due to the type definitions not being very well covered, but the other unit tests in the console repo call the queries defined in this file.

Reply:
OK, I see.